packages/core/src/components/ModalNew/ModalHeader/ModalHeader.module.scss

```diff
@@ -0,0 +1,14 @@
+.header {
+  width: 100%;
+  align-self: flex-start;
+
+  .title {
+    width: 100%;
```

Comment:
Does the title need to leave space for the header actions to guarantee that it won't overlap as it is absolute?

Reply:
Every layout would be responsible to manage the spacing eacb component needs, so the inline end would have padding on basic layout only

Reply:
Got it